packages/back-office-subscribers/nsip-service-user/index.js

```diff
@@ -6,8 +6,7 @@ module.exports = async (context, message) => {
 	const serviceUserId = message.id;
 
 	if (!serviceUserId) {
-		context.log(`skipping update as serviceUserId is missing`);
-		return;
+		throw new Error(`serviceUserId is required`);
```

Comment:
throw new Error(`id is required`); might be more clear as the message payload uses `message.id`.